src/CMakeLists.txt

```diff
@@ -253,8 +254,8 @@ elseif (MSVC)
     set(CMAKE_CXX_FLAGS_MINSIZEREL     "/Os /Zc:inline ${CMAKE_CXX_FLAGS_MINSIZEREL}")
     set(CMAKE_CXX_FLAGS_RELEASE        "/O2 /Oi /Oy /Zc:inline ${CMAKE_CXX_FLAGS_RELEASE}")
     set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "/O2 /Oi /Zi ${CMAKE_CXX_FLAGS_RELWITHDEBINFO}")
-    set(LEAN_EXTRA_LINKER_FLAGS        "${LEAN_EXTRA_LINKER_FLAGS} /LTCG:INCREMENTAL")
```

Comment:
You need to remove /GL from CMAKE_CXX_FLAGS as well if you want to disable LTO.

Reply:
I see, thanks. We're slowly getting there, hopefully :) .